tests/contrib/celery/test_chained_task.py

```diff
@@ -0,0 +1,45 @@
+import os
+import re
+import signal
+import subprocess
+import time
+
+
+# Ensure that when we call Celery chains, the root span has celery specific span tags
+# The test_integration.py setup doesn't perfectly mimic the condition of a worker process running.
+# This test runs the worker as a side so we can check the tracer logs afterwards to ensure expected span results.
+# See https://github.com/DataDog/dd-trace-py/issues/11479
+def test_task_chain_task_call_task():
+    celery_worker_cmd = "ddtrace-run celery -A tests.contrib.celery.tasks worker -c 1 -l DEBUG -n uniquename1 -P solo"
+    celery_task_runner_cmd = "ddtrace-run python tests/contrib/celery/run_tasks.py"
+    worker_process = subprocess.Popen(
+        celery_worker_cmd.split(),
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+        preexec_fn=os.setsid,
+        close_fds=True,
+        cwd=str(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))),
```

Comment:
do you want to split and look at [-1] ?

Reply:
I ended up just updating this so that we run the test in the current directory where the test live, instead of the convoluted way of trying to go back to the `dd-trace-py` root directory.

```
    celery_worker_cmd = "ddtrace-run celery -A tasks worker -c 1 -l DEBUG -n uniquename1 -P solo"
    celery_task_runner_cmd = "ddtrace-run python run_tasks.py"

    # The commands need to run from the directory where this test file lives
    current_directory = str(os.path.dirname(__file__))

```

Resolving this.